README.md

```diff
@@ -147,6 +147,16 @@ client
     .pipe(fs.createWriteStream("~/video.np4"));
 ```
 
+If you want to request only partial data, you can specify the range in the options argument of the method:
```

Comment:
Could you change this to: "If you want to stream only part of the file, you can specify the `range` in the options argument:"

Reply:
Yep, sounds better